app/components/button.jsx

```diff
@@ -139,7 +140,7 @@ const buttonStyles = createUseStyles(theme => ({
     lineHeight: '1.5rem',
     textAlign: 'center',
     '&:focus': {
-      outline: `${theme.focusOutline}`,
+     // outline: `${theme.focusOutline}`,
```

Comment:
The button component is used in a lot of places besides signup. Taking out the focus would affect them all.  If (and I'm not sure) we need to change the focus style for signup, we need to do it in a way that's specific to signup. 

It is possible to pass a className from Signup to Button that overrides the focus.